app.js

```diff
@@ -58,27 +67,34 @@ for (var t = -1; t < 15; t++){
   } else {
     var time = (t - 6) + 'pm';}
```

Comment:
This is all a lot of logic for just needing to handle some hours you could put into an array